.github/workflows/pr_body_check_jira.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Check body for reference to jira
+on:
+  pull_request:
+    branches:
+      - release_**
+      - feature_**
+jobs:
+  pr-check:
+    if: github.repository_owner == 'ansible' && github.repository != 'awx'
```

Comment:
the `feature_` branches are presumed to be in the awx repository, so I say we kick feature branches out of scope. After all, I'm not concerned about PRs to a _base_ branch that is a feature branch, I only want this kind of check when trying to merge a feature branch.